internal/pkg/install/mount.go

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package install
+
+import (
+	"log"
+
+	"github.com/autonomy/talos/internal/pkg/blockdevice/probe"
+	"github.com/autonomy/talos/internal/pkg/constants"
+	"github.com/autonomy/talos/internal/pkg/mount"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+)
+
+// Mount discovers the appropriate partitions by label and mounts them up
+// to the appropriate mountpoint.
+// TODO: See if we can consolidate this with rootfs/mount
+func Mount() (err error) {
+	log.Println("Discovering mountpoints ( sleep ) ")
```

Comment:
```suggestion
	log.Println("Discovering mountpoints")
```

Reply:
This is really the only nit I'd like to see fixed before merging, we had this here when debugging IIRC.